app/views/templates/_form.html.erb

```diff
@@ -106,12 +134,23 @@
         $j('#attribute-table').on('change', '.sample-type-attribute-type', SampleTypes.attributeTypeChanged);
         $j('#attribute-table').on('change', '.controlled-vocab-selection', SampleTypeControlledVocab.controlledVocabChanged);
         $j('#attribute-table .sample-type-attribute-type').trigger("change", [false]);
-				$j('.templates').select2({theme: "bootstrap"});
-				initSelect2($j('.templates'), $j('#existing_templates'))
+        $j('.templates').select2({theme: "bootstrap"});
```

Comment:
would be good, for consistency and future maintainabiliy, if in the future this could be refactored to use the `objects_input.js` or better the `objects_input ` helper. But I don't think it should be a blocker for this PR

Reply:
Good point. I made an issue (https://github.com/seek4science/seek/issues/1622) for this.